server-adapters/resteasy-reactor-netty/src/test/java/org/jboss/resteasy/test/NettyTest.java

```diff
@@ -0,0 +1,317 @@
+      Response headResponse = target.request().build(HttpMethod.HEAD).invoke();
+      // [AG] Discuss with @crankydillo.  The Netty test seems to be doing the same.
+      // were you expecting something else?
+      // TODO IDE passes the following assertion.  I think because it's
+      // confused about what reactor-netty and/or netty library is on the
+      // classpath.
+      Assert.assertNull(headResponse.getHeaderString("Content-Length"));
+      Assert.assertNull(headResponse.getHeaderString("transfer-encoding"));
+   }
+
+   @Test
+   public void testBasic() throws Exception
+   {
+      WebTarget target = client.target(generateURL("/test"));
+      String val = target.request().get(String.class);
+      Assert.assertEquals("hello world", val);
+   }
+
+   @Test
+   public void testQuery() throws Exception
+   {
+      WebTarget target = client.target(generateURL("/query"));
+      String val = target.queryParam("param", "val").request().get(String.class);
+      Assert.assertEquals("val", val);
+   }
+
+   @Test
+   public void testEmpty() throws Exception
+   {
+      WebTarget target = client.target(generateURL("/empty"));
+      Response response = target.request().get();
+      try
+      {
+         Assert.assertEquals(204, response.getStatus());
+      }
+      finally
+      {
+         response.close();
+      }
+   }
+
+   @Test
+   public void testLarge() throws Exception
+   {
+      WebTarget target = client.target(generateURL("/large"));
+      Response response = target.request().get();
+      try
+      {
+         Assert.assertEquals(200, response.getStatus());
+         StringBuffer buf = new StringBuffer();
+         for (int i = 0; i < 1000; i++) {
+            buf.append(i);
+         }
+         String expected = buf.toString();
+         String have = response.readEntity(String.class);
+         Assert.assertEquals(expected, have);
+
+      }
+      finally
+      {
+         response.close();
+      }
+   }
+
+   @Test
+   public void testUnhandledException() throws Exception
+   {
+      WebTarget target = client.target(generateURL("/exception"));
+      Response resp = target.request().get();
+      try
+      {
+         Assert.assertEquals(500, resp.getStatus());
+      }
+      finally
+      {
+         resp.close();
+      }
+   }
+
+   // [AG] Discuss with @crankydillo.  I see that Netty server pushes it to
+   // ResteasyContext, so that one could do `@Context ChannelHandlerContext context`.
+   // Please see https://github.com/resteasy/Resteasy/blob/4.5.8.Final/server-adapters/resteasy-netty4/src/main/java/org/jboss/resteasy/plugins/server/netty/RequestDispatcher.java#L83
+   // However, I am not sure if we need that for Reactor Netty.  I suggest
+   // we delete this test for now, and if a need arises in the future,
+   // we can consider adding the feature.
+   // TODO we do not set 'ChannelHandlerContext' (see resource method)
+   // Are we supposed to be doing something there?
+   //@Test
```

Comment:
Minor nit, could we keep the `@Test` here and add `@Ignore` please? Or we could just delete it.

Reply:
Not minor to us!  I almost certainly will remove this.